### ISSUE TITLE
some problems

### ISSUE THREAD

**Issue by GUJI1JIUJIU1UU1** [mdyfrank/UPRTH#1]
Hello, I met some problems when I ran evaluator/cpp/evaluate_foldout.py. It seems that it cannot find apt_evaluate_foldout.pyx in the same folder.

![image](https://github.com/mdyfrank/UPRTH/assets/119782418/28be0522-5631-415e-ac47-c727c3896ee0)
And I also failed to run setup.py.  Could you tell me are there any parameter commands that I should input?

![image](https://github.com/mdyfrank/UPRTH/assets/119782418/3136acef-6431-43f7-a3d7-05256021bc91)

**Comment by mdyfrank** [comment]
Yes you may need to run setup.py first to import apt_evaluate_foldout. But currently i am not sure why you have such error for setup.py. Have you installed cython in your environment?
